### ISSUE TITLE
Add support for thumbnails to MediaLibraryItems

### ISSUE THREAD

**Issue by tplessis** [octobercms/october#1535]
Hi guys,

Is there a way to get thumbnail for a media manager image, like for database images ? Like this : 

```
{{ product.featured_images[0].thumb(300, 300, { extension: 'png' }) }}
```

Regards,
Thomas

**Comment by danieljaniga** [comment]
+1

**Comment by hitch90** [comment]
+1

**Comment by odorisioe** [comment]
+1

**Comment by jamiegrand** [comment]
+1

**Comment by hitch90** [comment]
It works :) i used .getThumb(width, height, mode)

**Comment by tplessis** [comment]
@hitch90 : no, it doesn't for media manager images...

**Comment by jwilson8767** [comment]
+1

**Comment by murtin** [comment]
+1

**Comment by manogi** [comment]
+1

**Comment by zdavis** [comment]
+1

**Comment by manogi** [comment]
I made a plugin for this which is currently awaiting approval. You can find it here if you want to try it out:
https://github.com/manogi/october-mediathumb

**Comment by gregoryloichot** [comment]
+1

**Comment by neilcarpenter1** [comment]
+1

**Comment by LukeTowers** [comment]
The support demand for October is increasing, while the available time the founders have to give remains the same. These issues will be handled in time, however there are ways you can help:

- Submit a pull request that introduces this feature and wait for it to be reviewed
- [Post a bug bounty](https://www.bountysource.com/teams/october) for this specific issue so that more people can become involved with creating a fix
- [Make a financial contribution to the project](http://octobercms.com/fundraising) so we can allocate more time towards making improvements like these that benefit everyone

**Comment by LukeTowers** [comment]
I'll implement support for this as a part of the upcoming media library metadata improvements.

**Comment by daftspunk** [comment]
This was implemented recently and is available in version 1.1
